### PR TITLE
This script allow you to test a LDAP configuration from the Jenkins console

### PR DESCRIPTION
**Pull request by kuisathaverat**
@reviewbybees @cloudbees/team-support

### CODE REVIEW

File: GroupsFromLDAP.groovy [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import javax.naming.ldap.*
+import jenkins.model.Jenkins;
+import nectar.plugins.rbac.strategy.*;
+import hudson.security.*;
+import nectar.plugins.rbac.groups.*;
+import nectar.plugins.rbac.roles.*;
+
+
+try {
+        String ldapAdServer = "ldap://192.168.1.36:389"
```
Comment (christ66) [inline GroupsFromLDAP.groovy]: ?

File: GroupsFromLDAP.groovy [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        String ldapSearchBase = "dc=example,dc=com"
+
+        String ldapUsername = "CN=tesla,CN=Users,DC=example,DC=com"
+        String ldapPassword = "Password12"
```
Comment (christ66) [inline GroupsFromLDAP.groovy]: ?!

File: GroupsFromLDAP.groovy (hunk 1 above)
Comment (kuisathaverat) [inline GroupsFromLDAP.groovy]: variable to store LDAP server URL

File: GroupsFromLDAP.groovy (hunk 2 above)
Comment (kuisathaverat) [inline GroupsFromLDAP.groovy]: rest of the paremeters to access to LDAP server

**Comment by stephenc** [comment]
@kuisathaverat ping

**Comment by kuisathaverat** [comment]
@christ66 @stephenc I changed the names to follow RFC recommendations , Is there anything else to solve?

**Comment by stephenc** [comment]
:bee: from me, but you are still blocked by @christ66

File: GroupsFromLDAP.groovy [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+import javax.naming.ldap.*
+import jenkins.model.Jenkins;
+import nectar.plugins.rbac.strategy.*;
+import hudson.security.*;
+import nectar.plugins.rbac.groups.*;
+import nectar.plugins.rbac.roles.*;
+
+
+try {
+        String ldapAdServer = "ldap://192.0.2.36:389"
```
Comment (carlosrodlop) [inline GroupsFromLDAP.groovy]: Just a suggestion "ldap://192.0.2.36:389" could be changed for ldap://ldap.yourcompany.com:389 to keep consistency with https://wiki.jenkins-ci.org/display/JENKINS/LDAP+Plugin

File: GroupsFromLDAP.groovy [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        String ldapSearchBase = "dc=example,dc=com"
+
+        String ldapUsername = "CN=tesla,CN=Users,DC=example,DC=com"
+        String ldapPassword = "Password12"
```
Comment (carlosrodlop) [inline GroupsFromLDAP.groovy]: For the same principle that the above comment, as a suggestion "ldapUsername" and "ldapPassword" could be changed for  "ldapManagerDN" and "password"

**Comment by kuisathaverat** [comment]
@christ66 Could you review this PR again after the changes to remove the 🐛 ?

**Pull request merged**